src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -123,6 +123,21 @@ Copyright (c) Microsoft Corporation. All rights reserved.
 
 https://aka.ms/pscore6-docs
 Type 'help' to get help.</value>
+  </data>
```

Comment:
I don't like that we have a copy of the syntax.  If we make updates in the future, we need to manually ensure that both sections are updated.  Perhaps we can have the syntax (aka short help) just called `HelpSyntax` and the other one called `ExtendedHelp`.  Now change the code where full help would have been shown shows `HelpSyntax` followed by `ExtendedHelp`.

Reply:
If `ExtendedHelp` maybe second - `UsageHelp` or `ShortHelp`?

Reply:
`UsageHelp` sounds fine to me

Reply:
Fixed.